src/platforms/common/configuration/options.mdx

```diff
@@ -411,6 +411,30 @@ If the callback is not set, or it returns `undefined`, the default naming scheme
 
 </PlatformSection>
 
+<ConfigKey name="send-client-reports" supported={["java", "javascript", "dotnet", "python", "dart", "apple"]}>
+
+Set this boolean to `false` to disable sending of client reports.
```

Comment:
I'd rather explain shortly what the client reports is about, every other property has a short description, otherwise, the user knows that it exists but it does not know its meaning.
Maybe @imatwawana can help here or copy-pasta from https://develop.sentry.dev/sdk/client-reports/

Reply:
Added something from there.